bin/fetchdata.js

```diff
@@ -148,7 +161,7 @@ function getUniqueCards(cards) {
     playableCardTitles.sort();
 
     const uniqueCards = [...uniqueCardsMap].map(([internalName, card]) => card);
-    return { uniqueCards, cardMap, playableCardTitles, duplicatesWithSetCode };
+    return { uniqueCards, cardMap, playableCardTitles, duplicatesWithSetCode, setCodeMap };
```

Comment:
Do you think it would be worth sorting the map entries by set and number to improve human readability? TS maps keep insertion order so it's doable by converting to an array and then back to a map, just not sure if we think there's much benefit